crates/ide/src/hover/tests.rs

```diff
@@ -8465,7 +8465,7 @@ impl Iterator for S {
                                 file_id: FileId(
                                     1,
                                 ),
-                                full_range: 7800..8008,
+                                full_range: 7800..8042,
```

Comment:
We really need to do something about these tests its so annoying that touching minicore keeps changing this